Source/Core/Core/PowerPC/Jit64/Jit_Integer.cpp

```diff
@@ -1525,7 +1525,7 @@ void Jit64::rlwimix(UGeckoInstruction inst)
 			bool isRightShift = mask == (1U << inst.SH) - 1;
 			if (gpr.R(a).IsImm())
 			{
-				u32 maskA = gpr.R(a).offset & ~mask;
+				u32 maskA = (u32)gpr.R(a).offset & ~mask;
```

Comment:
I'm not sure about this one.

Either OpArg.offset should be a u32, or we are ignoring a case where offset is larger than a u32.